DocumentedExamples/Querying_Scalar_Quantities_and_Annually_Averaged_Timeseries.ipynb

```diff
@@ -0,0 +1,2091 @@
+{
```

Comment:
<p>What is going on with <code>025deg_jra55_iaf_omip2_cycle6_78to08</code>? There is a gap from 1968 to 2010.</p><p> </p><p>I don't know what the history of the <code>025deg_jra55_iaf_omip2_cycle6_ext</code>,<code>*_14to17</code> and <code>*78to08</code> runs are. However, I suggest they be dropped from this comparison as they are confusing. The 6 cycles are sufficient for this demonstration.      </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>